docs-chef-io/content/server/auth.md

```diff
@@ -386,6 +386,61 @@ common causes:
     minutes. This can be fixed by syncing the clock with an Network Time
     Protocol (NTP) server.
 
+## User public key update
+
+Chef infra server user can update public_key using below methods.
```

Comment:
```suggestion
Chef infra server user can update public_key using below methods.
```
Should it be public_key or public key?  I don't know the answer, I just noticed that elsewhere here you use public key, not public key.